tests/memberclient.py

```diff
@@ -40,39 +40,38 @@ def test_missing_signature_header(network, args):
 
 
 def make_signature_corrupter(fn):
-    class SignatureCorrupter(ccf.clients.HTTPSignatureAuth_AlwaysDigest):
-        def __call__(self, request):
-            r = super(SignatureCorrupter, self).__call__(request)
-            return fn(r)
+    class SignatureCorrupter(ccf.clients.HttpSig):
+        def auth_flow(self, request):
+            yield fn(next(super(SignatureCorrupter, self).auth_flow(request)))
 
     return SignatureCorrupter
 
 
-signature_regex = 'signature="([^"]*)",'
+signature_regex = 'signature="([^"]*)"'
 
 
 def missing_signature(request):
-    original = request.headers["Authorization"]
-    request.headers["Authorization"] = re.sub(signature_regex, "", original)
+    original = request.headers["authorization"]
```

Comment:
Does httpx not use case-insensitive headers?

Reply:
It does, but since we use lower case everywhere for signing, it seemed weird not to here. I don't mind changing it back if you prefer.

Reply:
No that's fine, just wanted to check.